app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+class Post < ApplicationRecord
+  has_many :comments
+  belongs_to :author, class_name: 'User'
+
+  after_create :increment_posts_counter
+
+  def increment_posts_counter
+    author.increment!(:posts_counter)
+  end
+
+  def five_recent_comments
+    comments.order(created_at: :desc).first(5)
+  end
+end
```

Comment:
- [ ] a post should be able to have many likes as well, similar to the comments.